README.md

```diff
@@ -1554,6 +1557,7 @@ Table of Contents
   * [Dev.to](https://dev.to/) - Where programmers share ideas and help each other grow.
   * [Hashnode](https://hashnode.com/) — Hassle-free Blogging Software for Developers!.
   * [Medium](https://medium.com/) — Get smarter about what matters to you.
+  * [devRant](https://devrant.com/) - Share and bond over successes and frustrations with code, tech and life as a programmer.
```

Comment:
This doesn't seem to be a blogging platform, its more like reddit for developers.

Reply:
Thanks, I removed SOAP UI, devRant and Taurus.
I will come up with other SaaS in the next update (maybe next month). Thanks